### PR TITLE
More modules added to pendulum example

### PR DESCRIPTION
**Pull request by oluowoj**
- Datadefs, TMods, GenDefs, Assumptions added.
- Still need to fill in the blanks in introduction section.
- Derivations of GenDefs still need to be added
- IMods not in yet.

### CODE REVIEW

**Comment by oluowoj** [comment]
I can't seem to figure out the cause of the faliure. @JacquesCarette any clue?

**Comment by oluowoj** [comment]
Here is the error from the last build:
image

**Comment by smiths** [comment]
I've started looking at the SRS for the pendulum.  There are some odd lines in the text, such as `pendulumTitle is the subject pendulumTitle is the focus. Pendulum.` in the Introduction.  Maybe this is because this is a work in progress.

The derivations of the equations, like for the general definitions, are missing.  Is this something that is still to be added?

**Comment by JacquesCarette** [comment]
If you actually open that section in the build report, then scroll down (a lot), the error is right there:

>    LaTeX Warning: File `../../../datafiles/DblPendulum/pendulum.png' not found on input line 160.
> 
> ! Package pdftex.def Error: File `../../../datafiles/DblPendulum/pendulum.png'  not found.

**Comment by oluowoj** [comment]
> If you actually open that section in the build report, then scroll down (a lot), the error is right there:
> 
> > LaTeX Warning: File `../../../datafiles/DblPendulum/pendulum.png' not found on input line 160.
> > ! Package pdftex.def Error: File `../../../datafiles/DblPendulum/pendulum.png'  not found.

Oh hmm..., I thought I fixed this one. I will check the file again. Thanks

**Comment by oluowoj** [comment]
Placeholder info in SRS will be fixed with next PR. I just wanted to make sure the build was successful then I can make the minor changes.

**Comment by oluowoj** [comment]
> If you actually open that section in the build report, then scroll down (a lot), the error is right there:
> 
> > LaTeX Warning: File `../../../datafiles/DblPendulum/pendulum.png' not found on input line 160.
> > ! Package pdftex.def Error: File `../../../datafiles/DblPendulum/pendulum.png'  not found.

Error has been resolved

**Comment by oluowoj** [comment]
waiting on this to continue. Please merge

**Comment by oluowoj** [comment]
@JacquesCarette, merge

**Comment by oluowoj** [comment]
Leave branch undeleted

**Comment by oluowoj** [comment]
waiting for this branch to be merged please

**Comment by JacquesCarette** [comment]
Sorry - should be able to get to it today.

File: code/drasil-data/Data/Drasil/Concepts/Math.hs
Comment (JacquesCarette) [inline code/drasil-data/Data/Drasil/Concepts/Math.hs:85]: It would be nice to keep this lined up (that can be fixed later)

File: code/drasil-example/Drasil/DblPendulum/Body.hs
Comment (JacquesCarette) [inline code/drasil-example/Drasil/DblPendulum/Body.hs:90]: `pendulumTitle` in the string? I think this should be fixed before merging.

File: code/drasil-example/Drasil/DblPendulum/Body.hs
Comment (JacquesCarette) [inline code/drasil-example/Drasil/DblPendulum/Body.hs:94]: same here.

File: code/drasil-example/Drasil/DblPendulum/References.hs
Comment (JacquesCarette) [inline code/drasil-example/Drasil/DblPendulum/References.hs:11]: If this is not used / useful, please delete these comments.

File: code/drasil-example/Drasil/DblPendulum/IMods.hs
Comment (JacquesCarette) [inline code/drasil-example/Drasil/DblPendulum/IMods.hs:99]: Please make sure that the only commented out things that are kept are useful.

File: code/stable/dblpendulum/SRS/DblPendulum_SRS.tex
Comment (JacquesCarette) [inline code/stable/dblpendulum/SRS/DblPendulum_SRS.tex:134]: This sentence is not well-formed. And why is there a `Pendulum..` at the end?

**Review by JacquesCarette — CHANGES_REQUESTED** [review]

File: code/drasil-example/Drasil/DblPendulum/Body.hs
Comment (oluowoj) [reply code/drasil-example/Drasil/DblPendulum/Body.hs:90]: done

File: code/drasil-example/Drasil/DblPendulum/Body.hs
Comment (oluowoj) [reply code/drasil-example/Drasil/DblPendulum/Body.hs:94]: done

File: code/stable/dblpendulum/SRS/DblPendulum_SRS.tex
Comment (oluowoj) [reply code/stable/dblpendulum/SRS/DblPendulum_SRS.tex:134]: done

**Comment by oluowoj** [comment]
still getting errors with CI Tex
https://travis-ci.com/github/JacquesCarette/Drasil/builds/202587251

**Comment by oluowoj** [comment]
@carette, could you please merge and delete branch?

**Review by JacquesCarette — APPROVED** [review]

**Pull request merged**
